common/Storyboarding/OsbSprite.cs

```diff
@@ -15,6 +15,8 @@ public class OsbSprite : StoryboardObject
         private List<ICommand> commands = new List<ICommand>();
         private CommandGroup currentCommandGroup;
 
+        public int MaxCommandCount = 300;
```

Comment:
default should probably be 0 = don't split

Reply:
makes sense, done